package.json

```diff
@@ -102,7 +102,7 @@
     "clean-install": "yarn clean && yarn",
     "start": "webpack serve --config webpack.dev.js",
     "analyze": "webpack --config webpack.analyze.js",
-    "build": "rimraf dist/ && mkdirp dist && cp README.md dist && rollup -c && webpack",
+    "build": "rimraf dist/ && mkdirp dist && cp README.md dist && rollup -c && webpack && node update-sri.js package dist/README.md",
```

Comment:
Is there a reason `dist` doesn't have `contrib/*/README.md`?

Reply:
@edemaine I'm not sure but since they are hosted online, I think only the root README is enough.